### PR TITLE
Add CentOS 7 to CI

### PR DESCRIPTION
**Pull request by superm1**

### CODE REVIEW

**Comment by YehezkelShB** [comment]
(Small typo in the 2nd commit message: sytemd -> systemd)

**Comment by superm1** [comment]
> (Small typo in the 2nd commit message: sytemd -> systemd)

Thanks, fixed.

File: src/meson.build [hunk 1]
```diff
@@ -16,6 +17,7 @@ endif
 if get_option('pkcs7')
   keyring_src += 'fu-keyring-pkcs7.c'
   keyring_deps += gnutls
+  test_deps+= colorhug_pkcs7_signature
```
Comment (hughsie) [inline src/meson.build]: Space before the += pls

**Comment by hughsie** [comment]
A tiny nit, and then LGTM, thanks!

**Pull request merged**